xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1781,6 +1781,8 @@ void CVideoPlayer::ProcessRadioRDSData(CDemuxStream* pStream, DemuxPacket* pPack
 
 bool CVideoPlayer::GetCachingTimes(double& level, double& delay, double& offset)
 {
+  static bool cache_low_level = true;
```

Comment:
Yes, static won't work with multiple instances of VideoPlayer. ProcessInfo is for collecting data at runtime and make VideoPlayer act according to the data. If you think this caching data is something VideoPlayer should change behaviour if those values change, you could add it to ProcessInfo. I'd appreciate if you could do if you have time and want to look into it. But I am also comfortable if you turn it into a member.